tests/components/acmeda/test_cover.py

```diff
@@ -0,0 +1,40 @@
+"""Define tests for the Acmeda config flow."""
+
+import pytest
+
+from homeassistant.components.acmeda.const import DOMAIN
+from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.setup import async_setup_component
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_config_entry(hass: HomeAssistant) -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    mock_config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={},
+    )
+    mock_config_entry.add_to_hass(hass)
+    return mock_config_entry
+
+
+async def test_cover_id_migration(
+    hass: HomeAssistant, mock_config_entry: MockConfigEntry
+) -> None:
+    """Test migrating unique id."""
+    mock_config_entry.add_to_hass(hass)
+    entity_registry = er.async_get(hass)
```

Comment:
Entity registry can be loaded in as test fixture

Reply:
Done